### PR TITLE
Fixes for exif in heic files from my Note10+

### PR DESCRIPTION
**Pull request by drewp**
Sorry, I don't have convenient sharable sample images. Or test code.

The error you get without this fix is like this:

exifread/heic.py:229:find_exif
exifread/heic.py:123:expect_parse

<class 'exifread.heic.WrongBox'>: ('meta', 'mdat')

### CODE REVIEW

**Comment by ianare** [comment]
Hi, thanks for the PR.

There's a problem with the tests though, the code is not compatible with Python 2.7 and 3.5.

https://travis-ci.org/github/ianare/exif-py/builds/737348143

**Comment by drewp** [comment]
The print() line is an accident, but it took me like 4 tries to get this to be a PR at all. Does anyone else want to extract the print and make a new PR? I may not get around to it soon.

File: exifread/heic.py [hunk 1]
```diff
@@ -49,6 +49,9 @@ def get(self, nbytes):
         read = self.file_handle.read(nbytes)
         if not read:
             raise EOFError
+        if len(read) != nbytes:
+            print(f'current pos={self.file_handle.tell()}')
+            raise ValueError(f'get(nbytes={nbytes}) found {len(read)} bytes')
```
Comment (ianare) [inline exifread/heic.py:54]:
```suggestion
            msg = "get(nbytes={nbytes}) found {read} bytes at postion {pos}".format(
                nbytes=nbytes,
                read=len(read),
                pos=self.file_handle.tell()
            )
            raise BadSize(msg)
```

Comment (ianare) [reply exifread/heic.py:54]: The f-string format was introduced in Python 3.6 which is why there is a syntax error on previous versions.
Using the `BadSize` exception specific to this class. The exception should get printed by the exception handling mechanism so no need to have a separate print statement.

**Pull request merged**